src/nwsapi.js

```diff
@@ -1162,7 +1162,7 @@
                     'if((' +
                       '((/^textarea$/i.test(e.localName)&&!e.readOnly&&!e.disabled)||' +
                       '("|date|datetime-local|email|month|number|password|search|tel|text|time|url|week|".includes("|"+e.type+"|")&&!e.readOnly&&!e.disabled))||' +
-                      '(e.hasAttribute("contenteditable")||(s.doc.designMode=="on"))' +
+                      '((e.hasAttribute("contenteditable")&&e.getAttribute("contenteditable")!="false")||(s.doc.designMode=="on"))' +
```

Comment:
I don't see a reason to keep the `hasAttribute("contenteditable")` call

Reply:
Good catch, updated.